resources/arduino_files/telemetry_board/telemetry_board.ino

```diff
@@ -1,259 +1,603 @@
+/*
+   The PANOPTES Baseline Unit (January 2017) telemetry board controls 5 relays on
+   the V0 power board, and reads 4 current sensors, 3 temperature sensors and one
+   combined humidity & temperature sensor. Each pass through loop() it checks for
+   serial input regarding switching the relays, and every 2 seconds it prints
+   out the values of the sensors and the settings of the relays.
+
+   This program has a class for each type of sensor, with a common API:
+   1) An Init() method that is called from setup.
```

Comment:
Camera board has these methods as lower case names. We can make it conform in a PR that moves things to a shared library if you want.

Reply:
Good point. I'll clearly be forced to resolve that when extracting the shared classes.